app/views/devise/mailer/confirmation_instructions.html.haml

```diff
@@ -0,0 +1,10 @@
+%div(style='margin: auto; width: 600px; border: 1px solid grey; border-radius: 5px; padding: 20px; text-align: center')
```

Comment:
Why do you use inline css styles in your haml files?

Reply:
This is difficult to read and maintain. I suggest taking a look at this [gem](https://github.com/fphilipe/premailer-rails)
@okohuch what do you think?

Reply:
@vikabelous I agree. @anastasiabalaban let's use gem Vika suggested.